src/player.h

```diff
@@ -355,6 +355,8 @@ class player : public Character
         bool has_alarm_clock() const;
         /** Returns true if the player or their vehicle has a watch */
         bool has_watch() const;
+		/** Returns true if the player is illiterate */
```

Comment:
Tabs.

Reply:
Stupid VS, I thought I fixed all of these, but nope.